solutions/USACO feb21-clockwise-fence.mdx

```diff
@@ -9,7 +9,12 @@ author: Melody Yu
 
 ## Solution (Vector Cross Product)
 
-The vector cross product can be used to determine if 2 vectors are rotating clockwise or anticlockwise. For this question, we treat each step that Farmer John walks as a vector and take the cross product of the current step and the previous step. If we sum all the cross product values, we can determine if the overall direction is CW or CCW based on the sign of the sum.
+To find the area of a triangle with a vertex at (0, 0), a second vertex at (x1, y1) and third vertex at (x2, y2), we can use the formula (x1y2 - y1x2)/2. This value is positive if x1/y1 is greater than x2/y2, since the slope becomes smaller, making it clockwise. Likewise, a negative area means the direction is counterclockwise.
```

Comment:
```suggestion
To find the area of a triangle with a vertex at $(0, 0)$, a second vertex at $(x_1, y_1)$ and third vertex at $(x_2, y_2)$, we can use the formula $\frac{x_1y_2 - y_1x_2}{2}$. This value is positive if $\frac{x_1}{y_1}$ is greater than $\frac{x_2}{y_2}$, since the slope becomes smaller, making it clockwise. Likewise, a negative area means the direction is counterclockwise.
```

Reply:
this hasn't been fixed yet...